credoai/reporting/dataset_base.py

```diff
@@ -23,11 +24,34 @@ def create_report(self, filename=None):
         -------
         array of figures
         """
-        results_all = self.module.get_results()
+        self.results_all = self.module.get_results()
```

Comment:
Nit: not best practice to define instance variables outside of __init__. In the NLP reports I just get the results in each plot function. You can also just define this in the __init__

Reply:
Good idea. Applied.